instalation-arch-linux (#519)
timescale/docs: timescaledb/how-to-guides/install-timescaledb/instalation-arch-linux.md

## apt Installation (Ubuntu) [](installation-apt-ubuntu)

> This will install TimescaleDB via `pacman` on archlinux distros.

Reviewer: ```suggestion
This installs TimescaleDB using `pacman` on Arch Linux distributions.
```

**Note: TimescaleDB requires PostgreSQL 12 or 13.**

#### Prerequisites

> - archlinux X.X or later, except obsoleted versions.
> Check [archlinux.org/releng/releases/][archlinux-releases] for list of
> non-obsolete releases.

Reviewer: ```suggestion
You need an Arch Linux X.X or later ISO file, which you can download from 
[archlinux.org/releng/releases/][archlinux-releases]. Ensure the version of 
Arch Linux you want to install is not obsolete.
```

#### Build and install

<highlight type="warning">
TODO

> </highlight>

Reviewer: We can now do proper procedures, using `<procedure>` tags.

> Once you confirm and install the newest binary package, you must still perform the
> EXTENSION update as discussed in [Updating Timescale to 2.0][update-tsdb-2].

Reviewer: ```suggestion
When you have installed the latest binary package, you need to perform an EXTENSION
update, see [updating Timescale to 2.0][update-tsdb-2] for more information.
```

#### Configure your database

> There are a [variety of settings that can be configured][config] for your
> new database. At a minimum, you will need to update your `postgresql.conf`
> file to include `shared_preload_libraries = 'timescaledb'`.
> The easiest way to get started is to run `timescaledb-tune`...

Reviewer: ```suggestion
There are a [number of settings that you can configure][config] for your
new database. Make sure you update your `postgresql.conf`
file to include `shared_preload_libraries = 'timescaledb'`.
The easiest way to get started is to run `timescaledb-tune`...
```

> 
> This will ensure that our extension is properly added to the parameter
> `shared_preload_libraries` as well as offer suggestions for tuning memory,
> parallelism, and other settings.

Reviewer: ```suggestion
This ensures that the TimescaleDB extension is properly added to the 
`shared_preload_libraries` parameter, and can offer suggestions for 
tuning memory, parallelism, and other important settings.
```

> To get started you'll now need to restart PostgreSQL:

Reviewer: ```suggestion
To get started, restart PostgreSQL:
```

```bash

> # Restart PostgreSQL instance

Reviewer: ```suggestion
```
